### PR TITLE
feat: add vtable

### LINKED ISSUE #51: Supplement new library: `VTable`
Supplement new library: [`VTable`](https://github.com/VisActor/VTable)

### PR DESCRIPTION
**Pull request by fangsmile**
image

Fix #51

### CODE REVIEW

File: lib/features.ts [hunk 1]
```diff
@@ -108,6 +108,18 @@ export const Features = {
...
+    description:
+      "The library has built-in functionality which lets you export image.",
+    important: false,
+  },
+  treeHierarchy: {
+    title: "tree hierarchy",
+    description:
+      "The library has built-in functionality which can show struct data with tree hierarchy.",
+    important: false,
+  },
```
Comment (statico) [inline lib/features.ts:122]: I don't think we need these features. PDF and Word export ability seems more important than images. Also, I think the `treeHierarchy` feature you added is a duplicate of `trees` earlier in the file.

**Review by statico — CHANGES_REQUESTED** [review]
Thank you! I have a few requests regarding new library features, however.

File: lib/features.ts (hunk 1 above)
Comment (fangsmile) [reply lib/features.ts]: I have delete this features  @statico

**Comment by fangsmile** [comment]
@statico can you merge this pull request

**Comment by statico** [comment]
Sorry for the delay -- it's been a busy month. Thank you!

**Pull request merged**